Package.swift

```diff
@@ -48,6 +48,14 @@ let stubDataBuildSettings: [CXXSetting] = buildSettings.appending([
     .headerSearchPath("."),
 ])
 
+// Windows requires C++14 or newer as the C++ standard library is implemented
+// with the newer standard since MSVC defaults to that version currently.
+#if os(Windows)
+let CXXStandard: CXXLanguageStandard = .cxx14
```

Comment:
Should we just do this for all platforms? Any downside?

Reply:
Honestly, can't think of any.  I think it's a level of comfort thing.  clang definitely supports C++14, and main builds have adopted C++17 even, so this should be safe globally.